src/jqLite.js

```diff
@@ -284,7 +284,7 @@ function JQLite(element) {
   }
   if (!(this instanceof JQLite)) {
     if (argIsString && element.charAt(0) !== '<') {
-      throw jqLiteMinErr('nosel', 'Looking up elements via selectors is not supported by jqLite! See: http://docs.angularjs.org/api/angular.element');
+      return new JQLite(window.document.querySelector(element));
```

Comment:
This is not a good place to add this logic as it'd mean `new angular.element(selector)` wouldn't work. This whole `if` should just be removed and the logic should me moved down.

Reply:
Also, note it should use `querySelectorAll`, not `querySelector` as we want to select all matching elements, not one.